dipy/reconst/csdeconv.py

```diff
@@ -118,18 +120,20 @@ def __init__(self, gtab, response, reg_sphere=None, sh_order=8, lambda_=1,
         # scale lambda_ to account for differences in the number of
         # SH coefficients and number of mapped directions
         # This is exactly what is done in [4]_
-        self.lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
-                        (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.))
-                       )
+
+        lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
+                   (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.)))
```

Comment:
I don't see any benefit to setting $N_0=1$ at this stage, at least
$N_0=\sqrt{362.}$ has the benefit of backwards compatibility. If you
disagree with me, maybe we can discuss this in a separate issue.

Bago

On Tue, Jan 27, 2015 at 2:23 PM, Samuel St-Jean <notifications@github.com>
wrote:

> In dipy/reconst/csdeconv.py
> <https://github.com/nipy/dipy/pull/553#discussion_r23648699>:
>
> > @@ -118,18 +120,20 @@ def __init__(self, gtab, response, reg_sphere=None, sh_order=8, lambda_=1,
> >          # scale lambda_ to account for differences in the number of
> >          # SH coefficients and number of mapped directions
> >          # This is exactly what is done in [4]_
> > -        self.lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
> > -                        (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.))
> > -                       )
> > +
> > +        lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
> > +                   (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.)))
>
> Oh, so that would be in agreement with @arokem <https://github.com/arokem>
> previous comment then. Which would also mean that the sqrt(362) is just a
> hardcoded value without any underlying meaning either.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/553/files#r23648699>.
>